nucypher/blockchain/eth/__init__.py

```diff
@@ -14,12 +14,3 @@
 You should have received a copy of the GNU Affero General Public License
 along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
 """
-
-from nucypher.blockchain.eth.signers import Signer, ClefSigner, KeystoreSigner
-
-SIGNERS = {
-    ClefSigner.URI_SCHEME: ClefSigner,
-    KeystoreSigner.URI_SCHEME: KeystoreSigner,
-}
-
-Signer.SIGNERS = SIGNERS
```

Comment:
Great 👍

Reply:
Heh - it's still there but moved into a new package.